server/src/test/java/org/elasticsearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -264,11 +264,14 @@ public static void unblockNode(final String repository, final String node) {
         ((MockRepository)internalCluster().getInstance(RepositoriesService.class, node).repository(repository)).unblock();
     }
 
-    protected void createRepository(String repoName, String type, Path location) {
-        logger.info("-->  creating repository");
+    protected void createRepository(String repoName, String type, Settings.Builder settings) {
```

Comment:
Is it overriden somewhere? If not it can be static.

Reply:
It uses the `logger` instance which is not static in the tests (can't be because we want the test suit in the logger name).

Reply:
Right - did not spot that. Sorry for the noise.